product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/ScenarioTestBase.java

```diff
@@ -180,4 +194,33 @@ public Boolean call() throws Exception {
         };
     }
 
+    protected String getStringFromDocument(Document doc) throws TransformerException {
+        DOMSource domSource = new DOMSource(doc);
+        StringWriter writer = new StringWriter();
+        StreamResult result = new StreamResult(writer);
+        TransformerFactory tf = TransformerFactory.newInstance();
+        Transformer transformer = tf.newTransformer();
+        transformer.transform(domSource, result);
+        return writer.toString();
+    }
+
+    protected List<String> getListOfFiles(File filePath){
+        File[] listOfFiles = filePath.listFiles();
+        List<String> fileNames = new ArrayList<>();
+
+        for (File file:listOfFiles) {
+            if (file.isFile()) {
+                fileNames.add(file.getName());
+            }
+        }
+        return fileNames;
+    }
+
+    protected String getXmlContent (File folderName) throws ParserConfigurationException, IOException, SAXException, TransformerException {
```

Comment:
Shall we describe what this method does?

Reply:
This method was removed by [https://github.com/wso2/product-ei/pull/2853](https://github.com/wso2/product-ei/pull/2853)